index.js

```diff
@@ -0,0 +1,138 @@
+// First, make sure to make all buttons for caclautor,
+// meaning numbers, getTotal(),getHistory(), clear, undo,operants
+// replace cuurent js code with previous js code
+// Make a current operant and previous operant attribute  on screen, both equal to ""
+// for each number click, concatnate the value string of that number to the current operant
+// maybe use .innerHtml or something reaserch what you can use
+// then when an operant is clicked, cuurent operasant is now previous operant and
+// then modfiy current operant to be "". also save the operant that was clicked on a varaible
+// then when numbers are clicked, it should be concatnated to currentOperant.
+// then when equalSButton  is clicked, we done a switch staement to check which operant is used and
+// use corresponding function on calcualtor to calculate previousopprant, currentOperant with the operant
+// then call .getTotal()
+
+const screen = document.getElementById("screen");
+const numbers = document.querySelectorAll("[numbersToPress]");
+const operationButtons = document.querySelectorAll("#operationButtons");
+const equalsButton = document.querySelector("#equalsButton");
+const clearButton = document.querySelector("#clearButton");
+const previousAnsButton = document.querySelector("#previousAnswerButton");
+const historyButton = document.querySelector("#getHistory");
+let secondValue = "";
+let operant = "";
+let histroyShowing = true;
+let currentValue = 0;
+let operantCounter = 0;
+
+const calcualtor = new Calculator();
+
+function returnValueOfOpperation() {
+  if (screen.value === "" || screen.value === ".") return;
+
+  if (secondValue.includes(".")) {
+    currentValue = parseFloat(secondValue);
+  } else {
+    currentValue = parseInt(secondValue);
+  }
+
+  switch (operant) {
+    case "+":
+      calcualtor.add(currentValue);
+      screen.value = calcualtor.getTotal();
+      break;
+    case "-":
+      calcualtor.subtract(currentValue);
+      screen.value = calcualtor.getTotal();
+      break;
+    case "*":
+      calcualtor.multiply(currentValue);
+      screen.value = calcualtor.getTotal();
+      break;
+    case "/":
+      calcualtor.divide(currentValue);
+      screen.value = calcualtor.getTotal();
+      break;
+    case "^":
+      calcualtor.power(currentValue);
+      screen.value = calcualtor.getTotal();
+      break;
+    default:
+      return currentValue;
+  }
+}
+
+clearButton.addEventListener("click", () => {
+  document.getElementById("screen").value = "";
```

Comment:
You have the screen element in the screen variable, why are you getting it from the DOM again here?

Reply:
Good point. Changed it.